src/module/Job/context/JobContext.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable react-refresh/only-export-components */
+
+import React, { createContext, useState, useContext, useEffect } from "react";
+
+import type { Job } from "../../../types/Job";
+import { getJobs } from "../utils/httpClient";
+
+interface JobContextProps {
+  jobs: Job[];
+  setJobs: (data: Job[]) => void;
+}
+
+export const JobContext = createContext<JobContextProps>({
+  jobs: [],
+  setJobs: (_data: Job[]) => {},
```

Comment:
```suggestion
  setJobs: () => {},
```

здесь ты инициализируешь начальное состояние. Это не типизация контекста, а его инициализация. В инициализации типизировать не нужно